examples/CH4-refact-dpdispatcher/machine-ali-ehpc.json

```diff
@@ -0,0 +1,75 @@
+{
+  "api_version": "1.0",
+  "train": [
+    {
+      "command": "dp",
+      "machine": {
+        "batch_type": "PBS",
+        "context_type": "SSHContext",
+        "local_root": "./",
+        "remote_root": "/home/fengbo/work_path_dpdispatcher_test",
+        "remote_profile": {
+            "hostname": "39.xxx.xx.xx",
+            "username": "fengbo"
+        }
+      },
+      "resources": {
+        "number_node": 1,
+        "cpu_per_node": 4,
+        "gpu_per_node": 1,
+        "queue_name": "T4_4_15",
+        "group_size": 1,
+        "custom_flags":["#SBATCH --mem=32G"],
+        "strategy": {"if_cuda_multi_devices": true},
+        "para_deg": 3,
+        "source_list": ["/home/fengbo/deepmd.1.2.4.env"]
+      }
```

Comment:
Could you please add necessary explanations for key words, such as para_deg, custom_flags, like https://github.com/deepmodeling/dpgen/blob/devel/examples/machine/DeePMD-kit-1.x/machine-slurm-qe.json, or provide the link to dpdispatcher's manual, or update DP-GEN's manual's machine part?

Reply:
Maybe provide a link to dpdispatcher's manual could be better?  the explanations for these keys may be placed in a common place.